test/controllers/api/capabilities_controller_test.rb

```diff
@@ -7,27 +7,54 @@ class CapabilitiesControllerTest < ActionController::TestCase
     def test_routes
       assert_routing(
         { :path => "/api/capabilities", :method => :get },
-        { :controller => "api/capabilities", :action => "show" }
+        { :controller => "api/v06/capabilities", :action => "show", :api_version => "0.6" }
```

Comment:
Say we want to use semantic versioning like 1.2.0, how would this be reflected specifically in directory names like `v06`?

Reply:
@mmd-osm a reasonable assumption IMHO would be to only actually have different "directory" names for major versions (aka breaking changes), the client can then determine from the capabilities which minor version is actually supported and from that determine which backwards compatible features are present.